src/main/resources/logback.xml

```diff
@@ -23,14 +23,14 @@
             </appender>
         </else>
     </if>
-    <logger name="no.nav" level="${log.level.no.nav:-INFO}"/>
-    <logger name="no.nav.security.token" level="${log.level.no.nav.security.token:-WARN}"/>
+    <logger name="no.nav" level="${log.level.no.nav:-DEBUG}"/>
+    <logger name="no.nav.security.token" level="${log.level.no.nav.security.token:-DEBUG}"/>
     <logger name="jdbc.sql" level="${log.level.jdbc.sql:-ERROR}"/>
     <logger name="com.zaxxer.hikari" level="${log.level.com.zaxxer.hikari:-WARN}"/>
     <logger name="ch.qos.logback" level="${log.level.ch.qos.logback:-WARN}"/>
     <logger name="org.apache.cxf" level="${log.level.ch.qos.logback:-WARN}"/>
 
-    <root level="INFO">
+    <root level="DEBUG">
```

Comment:
Skal vi ha på debug logging når vi prodsetter også?

Reply:
En liten stund, ja. Stemmer for å skru det av etter en dag eller så. Gjør det forhåpentligvis lettere å feilsøke hvis noe tryner i prod.